command_line_env.env

```diff
@@ -0,0 +1,10 @@
+# Flask server configuration.
+FLASK_ENV=development
+FLASK_APP=/my-work-dir/todo_app/app
+  
+
+# Change the following values for local development.
+SECRET_KEY=secret-key
+DEVELOPER_API_KEY=2bf28cd9609b6d82c6accd4fa78be70c
+MY_SECRET=526dac3e066c92f0e02701bf983246e6a0aea2b065f516055e0f7e5098700caf
+SERVER_TOKEN=94027d7e2a6eedb95e62a24826ae1eff2d074717a7db7e1c0b108210813c3cfa
```

Comment:
You've accidentally pushed real credentials to GitHub.

I expected you to use the `.env` file that should already be present. If you want to add more .env files that contain real credentials for whatever reason, then it's best to add them to the `.gitignore` file so you don't accidentally commit them.

Since it's already being tracked, you'll need to run `git rm --cache README.md` to actually remove it.

Also since the credentials have already been leaked, you would normally want to rotate your credentials